### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by slava-viktorov** [tvooo/scope-inspector#15]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/usr/share/atom/resources/app/src/atom.js:54:11)
ScopeInspector.registerInspections (/home/slava/.atom/packages/scope-inspector/lib/scope-inspector.coffee:13:9)
```

**Comment by tvooo** [comment]
Thanks for the hint. Damned, I really need to update that plugin :)

**Comment by slava-viktorov** [comment]
do it! I like you plugin )

**Comment by tvooo** [comment]
Was fixed by 0586fa52042efda1aef0f0d795a071bd6ca4581e
